src/Schema/TypeRegistry.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema;
+
+use GraphQL\Type\Definition\Type;
+use Illuminate\Support\Collection;
+
+class TypeRegistry
+{
+    /**
+     * Collection of schema types.
+     *
+     * @var Collection
+     */
+    protected $types;
+
+    public function __construct()
+    {
+        $this->types = new Collection();
+    }
+
```

Comment:
Let's add a `instance` function that calls `get` to minimize breaking changes (goes along w/ adding the `schema` function back into the `helpers.php` file)

Reply:
Done!